packages/dashmate/docker-compose.platform.deps.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
```

Comment:
Instead of adding an additional service you can set the target as `${TARGET:-default}` and then build only this target with env variable TARGET=deps